overlord/snapstate/handlers.go

```diff
@@ -1902,6 +1916,19 @@ func (m *SnapManager) undoLinkSnap(t *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
+func countExcludedRevs(items []snap.Revision, seq []*snap.SideInfo) int {
+	var found int
+	for _, rev := range items {
+		for _, si := range seq {
```

Comment:
nitpick: this is not so important, since the function is very small, but I'd still suggest renaming `items` and `seq` to something that describes what they are, like `revisions` and `sideInfos`.